nautobot/ipam/tables.py

```diff
@@ -277,20 +282,25 @@ class VRFDeviceAssignmentTable(BaseTable):
     class Meta(BaseTable.Meta):
         model = VRFDeviceAssignment
         orderable = False
-        fields = ("vrf", "device", "virtual_machine", "rd", "tenant")
+        fields = ("vrf", "namespace", "device", "virtual_machine", "rd", "tenant")
 
 
 class VRFPrefixAssignmentTable(BaseTable):
     """Table for displaying VRF Prefix Assignments."""
 
     vrf = tables.Column(verbose_name="VRF", linkify=lambda record: record.vrf.get_absolute_url(), accessor="vrf.name")
+    namespace = tables.Column(
+        verbose_name="Namespace",
+        linkify=lambda record: record.vrf.namespace.get_absolute_url(),
+        accessor="vrf.namespace.name",
+    )
     prefix = tables.Column(linkify=True)
```

Comment:
Actually I'm wondering if this one (used only on the VRF detail view, which I don't see a screenshot of in the PR) is needed after all. Don't we enforce that Prefix/VRF assignments must always belong to the same common namespace? So won't the namespace be the same for every row in this table?

Reply:
Yeetus Deletus